app/views/refinery/shared/_navigation_language.html.erb

```diff
@@ -0,0 +1,10 @@
+<ul class="navigation-language">
```

Comment:
We should add hidden text before the ul for screen readers to indicate that this is the language selection:
```
<span class="hidden-text">Select language:</span>
```
```
.hidden-text {
  position: absolute;
  left: -1000px;
  top: auto;
  width: 1px;
  height: 1px;
  overflow: hidden;
}
```